website/docs/style_guide/about_style_guide.md

```diff
@@ -46,20 +46,20 @@ Here are some good basic guidelines for formatting your Robot Framework codes. A
 *** Keywords ***
 ```
 
-- Keywords within resource files should be alphabetized.
-- The recommended minimum line length is 120.
+- Keywords within resource files should be alphabetized or organized by functionality or purpose. The important part is to organize keywords in a thoughtful manner.
+- The recommended minimum maximum line length is 120.
```

Comment:
Are we sure we want to write "minimum maximum"? This is very confusing and the reader may think it was a mistake.

Reply:
yeah- we can just make the suggestion here; go into more detail in the specific guide.